src/connectionController.ts

```diff
@@ -165,26 +182,29 @@ export default class ConnectionController {
             return reject(new Error(`Failed to connect: ${error.message}`));
           }
 
-          const {
-            driverUrl,
-            instanceId
-          } = newConnectionConfig.getAttributes({
+          const { driverUrl, instanceId } = newConnectionConfig.getAttributes({
```

Comment:
I actually wanted to write about it here in the description, that I don't do it manually :) and sorry that real changes are being lost in automatic code prettifying. Lucas added settings file to the project and now vscode does it automatically. It reads settings from ` .vscode/settings.json`. I am surprised that it does not happen to you since the `settings.json` file is in git and you also use vscode for development, right?

Reply:
hmmm yup. I'll look into it ty